CHANGELOG-3.5.md

```diff
@@ -1,5 +1,9 @@
 # Changelog Lizmap 3.5
 
+## Unreleased
```

Comment:
Why this section?

Reply:
To prepare next release, the section Unreleased can contain the changelog of the next release.

Reply:
For now, we have one changelog file per version @rldhont so this change is not really needed.

It's not the best because difficult to reuse the content of these files. It's not following the convention https://keepachangelog.com/en/1.0.0/ but this is another topic.

Reply:
Your PR is for 3.5 so you should put description in 3.5 part.

Reply:
Sorry I was thinking 3.5.0 has been already released, I was fixing stable release.

Reply:
I have updated the changelog